src/main/java/stepdefs/CreateDefaultApplicationStepDefs.java

```diff
@@ -0,0 +1,53 @@
+package stepdefs;
+
+import businessobjects.User;
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import io.cucumber.datatable.DataTable;
+import org.testng.Assert;
+import pageobject.*;
+import services.LoginService;
+import util.DataGeneration;
+
+import java.util.List;
+
+public class CreateDefaultApplicationStepDefs {
+
+    @Given("^Login as Admin$")
+    public void loginAsAdmin() {
+        LoginService.loginToSF(new User());
```

Comment:
In case u do not want to split steps to PO class structure: 

this method return HomePage which is used in next test step

Create class variable 
private Homepage homepage;

update method

@Given("^Login as Admin$")
    public void loginAsAdmin() {
        homepage = LoginService.loginToSF(new User());
}

Please look through the rest methods and add more class variables like
clickApplicationsLinkInNavBar() -> ApplicationsPage 

Move to class variables only those which used in several method. If one method usage no need
like in method
    @When("^Populate first form with parameters:$")
    public void populateFirstFormWith(DataTable firstForm) {

NewApplicationFirstFormPage firstFormOfApplication is local and it is ok

Reply:
Done